sdk/src/azure/core/az_mqtt5_connection.c

```diff
@@ -7,7 +7,9 @@
 #include <azure/core/az_platform.h>
 #include <azure/core/az_result.h>
 #include <azure/core/az_span.h>
+#include <azure/core/az_retry.h>
 #include <azure/core/internal/az_log_internal.h>
+#include <azure/core/internal/az_retry_internal.h>
 #include <azure/iot/az_iot_common.h>
```

Comment:
I think this should be able to be removed now

Reply:
REmoved.

Reply:
```suggestion

```